manifests/ids.py

```diff
@@ -80,27 +83,36 @@ def main(data, document_id, source, host, cookie=None):
 #	structure_canvases = []
 
 	for cvs in data['response']['docs']:
+		if ( ('file_huldrsadmin_accessFlag_string' in cvs.keys()) and
```

Comment:
This might be nicer as since it means just one condition?

```python
if cvs.get('file_huldrsadmin_accessFlag_string') == 'N':
    continue  
```

Reply:
changed

Reply:
changed